test/unit/api/ready.js

```diff
@@ -0,0 +1,74 @@
+import fixture from '../../lib/fixture';
+import init from '../../../src/api/init';
+import ready from '../../../src/api/ready';
+import skate from '../../../src/index';
+import unique from '../../lib/element';
+
+describe('api/ready', function () {
+  let elem;
+  let tag;
+
+  function setup () {
+    skate(tag, {});
+  }
+
+  function initialise () {
+    init(elem);
+  }
+
+  beforeEach(function () {
+    tag = unique().safe;
+    elem = document.createElement(tag);
+  });
+
+  it('should fire for an element if no definitions are registered for it', function (done) {
+    ready(elem, function (shouldBeElem) {
+      expect(arguments.length).to.equal(1);
+      expect(shouldBeElem).to.equal(elem);
+      done();
+    });
+  });
+
+  it('should fire for an element when it is already ready', function (done) {
```

Comment:
It seems like this doesn't test the timing of the ready callback, it would be good to ensure the callback fires immediately once the element is ready. If possible doing something like this might be good:

    let readyCallbackFired = false;
    setup();
    ready(elem, function() {
       readyCallbackFired = true;
    });
    initialise();
    // wait a tick if we need to?
    expect(readyCallbackFired).to.equal(true);